js/client-chat.js

```diff
@@ -93,7 +93,8 @@
 			}
 			this.tabComplete.reset();
 			this.chatHistory.push(text);
-			text = this.parseCommand(text);
+			// make sure commands in multiline inputs are handled correctly
+			text = text.split('\n').map(this.parseCommand, this).filter(function (line) {return (line !== false);}).join('\n');
```

Comment:
Which part would be made easier by that change? It'd require re-adding the slash in order to parse the command, and adding a comment to explain why it's done like that. In terms of user experience, it makes client commands + text in a new line work in unexpected ways (like the current situation)